pyproject.toml

```diff
@@ -200,6 +200,11 @@ ray = [
 colab = [
   "pillow<=9.0.0",
 ]
+
+benchmarks = [
+  "datadog==0.45.0"
+]
```

Comment:
I would argue this is a requirement of the `test/benchmarks` run scripts, not Haystack's.